### ISSUE TITLE
2024 04 18 - GSF Netherlands - Launch Event 

### ISSUE THREAD

**Issue by Sophietn** [Green-Software-Foundation/meetup#147]
The GSF PM is responsible for updating this issue, ensuring the details section is up to date and the checklist marked based on the comments.

## Details

Introductions to green software principles and the Carbon Hack 24 hackathon experience.

@WBurggraaf please fill in the below details, and I'll get the group set up on the Meetup platform to create the first event

👥 Group: Link to meetup group
👉 Title: Proposed title
✍️ Organiser: Name | Affiliation | GitHub Handle
📆 Date: Date of the event
⏰ Time: From and To times (in local time)
📍 Venue: Where is the event going to be run?
 👥 Capacity: How many people can come?

## Checklist for Organiser

- [x] Date/Time for the event has been decided
- [x] Venue has been arranged
- [x] topic/format decided
- [x] speakers identified
- [ ] speaker bio/picture collected for Meetup.com & Social promotion
- [x] Catering arranged for event (please note, GSF cannot provide funding)
- [ ] Event Sponsors in place if applicable

## PM Checklist: 
- [x] Define event details: type, topic, audience (public, private, combo..)
- [x] Date locked? **before publishing the event**
- [x] Venue sourced? **before publishing the event**
- [x] Publish event on Meetup **at least 2 weeks prior the date**
- [x] Speaker sourced?

---

#### Meetup Page: 
- [x] Check organiser's permissions on Meetup
- [ ] Event description & settings
- [x] Event Cover requested from @jenya-design
- [ ] Include venue in MeetUp event
- [ ] Publish Agenda
- [ ] @namrata-woke notified for upcoming newsletters

#### Design Requirements
Meetup Event Card - please drop a comment below to @jenya-design including the following details
- [x] Location
- [x] Date/Time
- [ ] Hosting Organisation
- [x] Sponsor Logos
- [x] Title of the Event
- [ ] Any other details.

Here is an example of an event card: 
![image](https://github.com/Green-Software-Foundation/meetup/assets/20337337/92a4b076-81b9-4c42-896f-a484335fd7fb)

- [ ] If you require social assets, please add an additional comment.

#### Extra notes:
Please add this issue to the following Project Boards:
- [ ] **D: Graphic Design** if requesting assets
- [ ] **D: Marketing** for inclusion in Newsletters & GSF Social
- [ ] **Meetup** so it can be tracked as an event - _mandatory_

**Comment by Sophietn** [comment]
@WBurggraaf please provide the date name and venue so @Jenya-design can design the 1st event visual

**Comment by JeroenRaij** [comment]
@Jenya-design @Sophietn @WBurggraaf


**Event description:**
Welcome to the Dutch Green Software meetup network, organized by passionate software practitioners with the ambition to build a trusted ecosystem of people, standards, tooling and best practices for the development and use of green software.
Today, no organization can efficiently produce and market its product or service without the use of technology. Organizations that are able to support their core processes with well-functioning software remain relevant in this rapidly changing world.

From this, it is only a logical evolution to include the impact software has in reducing carbon emissions by using less physical resources, using less energy and using energy more intelligently. Making software ‘green’ must by a core priority, as important as software performance, security and cost.

The Green Software – the Netherlands group is for anyone with a passion for green software development and supports the Green Software Foundation (GSF) mission for sustainable software practices. We envision a thriving Dutch Green Software community empowered to take charge of its future.

Be welcome to join the community launch event and become part of this network! 

👥 Group: Link to meetup group
👉 Title: Community launch [live] event : Green Software the Netherlands
✍️ Organiser: HightTech Innovators & re:cinq
📆 Date: April 18th 2024
⏰ Time: 17:00 – 19:30
📍 Venue: HightTech Innovators, Rietveldenweg 40, 5222 AR ‘s-Hertogenbosch, Nederland
👥 Capacity: 60

**Comment by JeroenRaij** [comment]
hi @Sophietn @Jenya-design any idea whemn the community page & the event page are ready to use?

**Comment by Jenya-design** [comment]
@JeroenRaij @Sophietn @WBurggraaf
Please review the cover attached (option with and without GSF logo)
Editable Canva file [here](https://www.canva.com/design/DAF_fzpkFqA/B4sP9_Q4KqZXrgAqX-GpZQ/edit?utm_content=DAF_fzpkFqA&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton) (please make a copy before making any changes)
Let me know if you need any adjustments, I'll share the custom cover tomorrow

![Image](https://github.com/Green-Software-Foundation/meetup/assets/69306070/420ccae5-fdc4-46fe-8ec0-eb93f41d7bca)

![Image](https://github.com/Green-Software-Foundation/meetup/assets/69306070/b3b26312-ef2c-4e13-8cd9-2d5fcbffd17f)

**Comment by JeroenRaij** [comment]
@JeroenRaij @Sophietn @WBurggraaf @Jenya-design looks great! 

Is it possible to add also a 'old school' dutch windmill to it next to the modern one?  (left side of the image)

Lets use the set up nr 1

ps are you also making a cover for our community page itself? should be a similar image.

**Comment by Jenya-design** [comment]
@JeroenRaij thank you for review, I was planning to make 'old school' Dutch windmill vs modern ones for the community custom cover
as discussed in issue #146
Here's the example of the community custom cover for Türkiye 
https://github.com/Green-Software-Foundation/meetup/issues/133#issuecomment-1840997401
Similar style, but more detailed and different color accents to make it stand out

**Comment by JeroenRaij** [comment]
Ah OK! We can use the community cover for these and future events? As long as a traditional & a modern windmill are on the image, we can use these elements as our 'colour locale' :)

**Comment by Jenya-design** [comment]
@JeroenRaij I think for upcoming events it makes sense to reuse as a template the launch event cover I've shared today, as it has all the necessary text. I'll make the option with the windmill on the right

**Comment by JeroenRaij** [comment]
@Jenya-design  agreed! is this something we can update ourselves  in the future accordingly using the canva template?

**Comment by Jenya-design** [comment]
@JeroenRaij yes, you'll be able to update canva design as needed, just please make a copy first to keep the original version intact, it'll serve as a backup. Does that makes sense? You can also ask me to make the update if there's a need for considerable changes

**Comment by JeroenRaij** [comment]
@Jenya-design cool! if you make me the covwe image for the community page + for the launch event, then we have great templates to use for future purposes!

**Comment by Sophietn** [comment]
Hi both, just to confirm - the meetup group graphic won't change. This graphic is for the group and remains the same for all future events from that group. 
Then each event should have it's own graphic including date, and title (also possibly speakers, sponsors, venue)

**Comment by Jenya-design** [comment]
@JeroenRaij @WBurggraaf @Sophietn  please review and let me know if you need any adjustments (same published issue #146 )

Launch event cover (can be reused for other events, editable canva file [here](https://www.canva.com/design/DAF_fzpkFqA/B4sP9_Q4KqZXrgAqX-GpZQ/edit?utm_content=DAF_fzpkFqA&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)
![Netherlands-Meetup_local-event- 1200x676](https://github.com/Green-Software-Foundation/meetup/assets/69306070/8ff70701-db14-4637-a28c-daaff1a85b46)

Custom group cover
![Netherlands_Meetup group-01](https://github.com/Green-Software-Foundation/meetup/assets/69306070/9ca3e923-a30f-456c-9d08-fdda1f66719b)

**Comment by JeroenRaij** [comment]
@Jenya-design great job!

OK if i upload the custom group cover?

Thank you so much!

**Comment by JeroenRaij** [comment]
@Jenya-design  Maybe one remark :) the Netherlands has no hills, it is flat as a dime :). Can you do something bout that in the images? Both contain abstract hills.

**Comment by Jenya-design** [comment]
Thank you @JeroenRaij 
I would recommend to keep the hills for visual purposes, we need some action in otherwise static and gentle composition
I can smooth out some of the background hills for the group cover and send the update tomorrow evening
Does it work for you?

**Comment by JeroenRaij** [comment]
To be honest,  hill's are a bit silly for a country 10 feet below sea level ;)

We are known for our wind, coastline or clouds....is this something you can work with?



Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Sophietn** [comment]
Thank you @Jenya-design 

> OK if i upload the custom group cover?

@JeroenRaij  Yes you can upload it once it is ready. Once the group cover and the event cover are added, we can publicise the event.

**Comment by Jenya-design** [comment]
@JeroenRaij please review the update 

Launch event cover (canva file [here](https://www.canva.com/design/DAF_fzpkFqA/B4sP9_Q4KqZXrgAqX-GpZQ/edit?utm_content=DAF_fzpkFqA&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)
![Image](https://github.com/Green-Software-Foundation/meetup/assets/69306070/cf641cfb-808f-42b8-a08c-80602b99b0aa))

Custom group cover
![Image](https://github.com/Green-Software-Foundation/meetup/assets/69306070/68cd1588-baa5-47df-ae0a-4842be028f5f)
